tests/integration/records/expected_column_types.py

```diff
@@ -39,6 +39,16 @@
         "<class 'sqlalchemy.sql.sqltypes.DATETIME'>",
         "<class 'sqlalchemy.sql.sqltypes.TIMESTAMP'>"
     ],
+    # MySQL
```

Comment:
Same flag as above-- should these expectations be made explicit (e.g., that a certain path produces a certain output) be made explicit, since it would almost definitely be a bug if the type conversions changed for any particular path?

Reply:
I took a crack at this in https://github.com/bluelabsio/records-mover/pull/40